src/OrbitGl/PrimitiveAssembler.cpp

```diff
@@ -340,7 +340,7 @@ void PrimitiveAssembler::StartNewFrame() { batcher_->ResetElements(); }
 const orbit_client_protos::TimerInfo* PrimitiveAssembler::GetTimerInfo(PickingId id) const {
   const PickingUserData* data = GetUserData(id);
 
-  if (data && data->timer_info_) {
+  if ((data != nullptr) && (data->timer_info_ != nullptr)) {
```

Comment:
I don't mind them, but personally  I wouldn't have added the parentheses here and in some other places in this change where you added `!= nullptr`. I think it's normal to write `ptr != nullptr && ptr->foo`.

Reply:
Removed it.